src/modules/landing_target_estimator/LandingTargetEstimator.cpp

```diff
@@ -1,6 +1,6 @@
 /****************************************************************************
  *
- *   Copyright (c) 2013-2018 PX4 Development Team. All rights reserved.
+ *   Copyright (c) 2018-2022 PX4 Development Team. All rights reserved.
```

Comment:
```suggestion
 *   Copyright (c) 2013-2022 PX4 Development Team. All rights reserved.
```

:stuck_out_tongue:

Reply:
But you changed `2013-2018` to `2018-2022`. Isn't it missing the first five years then?

Reply:
The landing_target_estimator was added in 2018.

Reply:
Ah, I didn't check the actual git history :smile: